### PR TITLE
Add options to Teacher1819Application model

### PR DESCRIPTION
**Pull request by aoby**
[Task](https://trello.com/c/qyUyT5xw/242-submit-a-teacher-application)
[Spec](https://docs.google.com/document/d/1nb33WCCKy_PlU8IEeontJpsuwLvRYtq87Fxlsjyg6v8/edit?ts=5a00bf6b#)

### CODE REVIEW

File: dashboard/app/models/pd/application/application_base.rb [hunk 1]
```diff
@@ -36,7 +36,36 @@ class ApplicationBase < ActiveRecord::Base
     include ApplicationConstants
     include Pd::Form
 
+    OTHER = 'Other'.freeze
```
Comment (mehalshah) [inline dashboard/app/models/pd/application/application_base.rb:39]: Idea - a ruby gem that auto-freezes constants

File: dashboard/app/models/pd/application/teacher1819_application.rb [hunk 2]
```diff
@@ -34,5 +34,229 @@ def set_type_and_year
...
+        grades_teaching: [
+          *GRADES,
+          "I'm not teaching this year (please explain):"
+        ],
+        grades_expect_to_teach: GRADES,
+
+        subjects_teaching: SUBJECTS_THIS_YEAR,
+        subjects_expect_to_teach: SUBJECTS_THIS_YEAR,
+
+        subjects_licensed_to_teach: [
```
Comment (mehalshah) [inline dashboard/app/models/pd/application/teacher1819_application.rb:129]: Should this be the same as the other subject list?

File: dashboard/app/models/pd/application/teacher1819_application.rb [hunk 3]
```diff
@@ -34,5 +34,229 @@ def set_type_and_year
...
+        ],
+
+        taught_in_past: [
+          'Hour of Code',
+          'CS Fundamentals',
+          'CS in Algebra',
+          'CS in Science',
+          'CS Discoveries',
+          'CS Principles (intro or AP-level)',
+          'AP CS A',
```
Comment (mehalshah) [inline dashboard/app/models/pd/application/teacher1819_application.rb:155]: It hasn't been taught in 8 years but should AP CS AB be here?

Comment (tanyaparker) [reply dashboard/app/models/pd/application/teacher1819_application.rb:155]: Nope, we don't need to include.

File: dashboard/app/models/pd/application/teacher1819_application.rb (hunk 2 above)
Comment (tanyaparker) [reply dashboard/app/models/pd/application/teacher1819_application.rb:129]: It's intentional that this list is slightly different. Thanks!

**Pull request merged**